maintainers/maintainer-list.nix

```diff
@@ -5230,6 +5230,12 @@
     githubId = 9425955;
     name = "Jean-François Labonté";
   };
+  jeffBrown = {
```

Comment:
it is preferred that maintainer changes happen as a separate pr or commit, your call